### ISSUE TITLE
AP_GPS: Support reporting rtk-baseline distance for ublox (ZED-F9P)

### ISSUE THREAD

**Issue by ajshank** [ArduPilot/ardupilot#14965]
## Feature request

Ublox ZED-F9P and some other boards report a `RELPOSNED` message that contains information about the rover's distance to the RTK base-station. This information is picked up by Mavlink's `GPS_RTK` message [(Common, 127)](https://mavlink.io/en/messages/common.html#GPS_RTK). This is useful for operations relative to the base station (e.g., an offboard computer can adjust its map to a coordinate frame with origin at the base). Reporting this is currently supported only for the ERB driver in `master`.

**Solution**
While parsing the `RELPOSNED` message, also fill the corresponding states of interest in [UBLOX.cpp (L1388 .. L1403)](https://github.com/ajshank/ardupilot/blob/8cf0900064123b182b56e1640b4775d0d306f4e1/libraries/AP_GPS/AP_GPS_UBLOX.cpp#L1388), and also state that we support this mavlink message in [UBLOX.h (L147)](https://github.com/ajshank/ardupilot/blob/8cf0900064123b182b56e1640b4775d0d306f4e1/libraries/AP_GPS/AP_GPS_UBLOX.h#L147). This has been tested in multiple flights on the latest Copter with two ZED-F9P devices configured as base and rover, and parsed over in mavlink/mavros.

**Platform**
[  ] All
[  ] AntennaTracker
[ x ] Copter
[ x ] Plane
[ x ] Rover
[  ] Submarine

**Additional**
Happy to clean it up for a PR if that helps.

**Comment by amilcarlucas** [comment]
Yes, please do a PR. I did the ERB implementation and I will review it.
